src/chrome/content/rules/IslamWeb.net.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+	Refused:
+	- audio2.islamweb.net
+	- bwb.islamweb.net
+	- gk.islamweb.net (serves traffic from port 8080)
+	- kids.islamweb.net
+	- kidsen.islamweb.net
+	- quran.islamweb.net
+	- tube.islamweb.net
+
+-->
+<ruleset name="IslamWeb.net">
+	<target host="islamweb.net" />
+	<target host="www.islamweb.net" />
+	<target host="articles.islamweb.net" />
```

Comment:
This is common for *all* the HTTPS subdomains under this domain.  I think it's probably worth the tradeoff, though.  Accepting changes.